src/utils/browse.js

```diff
@@ -0,0 +1,21 @@
+import {
+  SHOW_LOAD_SPACE_PROMPT_CHANNEL,
+  RESPOND_LOAD_SPACE_PROMPT_CHANNEL,
+} from '../config/channels';
+
+// eslint-disable-next-line import/prefer-default-export
+export const showBrowsePrompt = (handleFileLocation) => {
```

Comment:
I think it is a too specific function (load only datasets) to be put in `utils`. In `graasp-desktop` we instead go for more specific listeners (a show prompt for each purpose) instead of a generic one triggered by specific "actions". So I would rather move `options` in the listener directly, and have `showBrowsePrompt` in the `LoadFile` component. Let me know what you think.

Reply:
Makes sense. I moved this code into the `handleBrowse` function in the component, and removed the intermediate `showBrowsePrompt` function. However, not sure what you mean by moving the `options` to the listener (?). I left it in `handleBrowse`.